tests/functional/spec/features/repeating_blocks/list_collector_repeating_blocks.spec.js

```diff
@@ -320,11 +321,36 @@ describe("Given a journey through the list collector with repeating blocks, in a
     await $(CompaniesRepeatingBlock2Page.authorisedTraderUkRadioYes()).click();
     await $(CompaniesRepeatingBlock2Page.submit()).click();
   });
-
-  it("The section can now be submitted and and hub completed.", async () => {
+  it("Given I add additional company via a list collector, When I edit newly added list item, Then I safely return to list collector page", async () => {
```

Comment:
This doesn't match the spec style or flow. The other `it` blocks are not `Given ..` styles.

I think we need to revisit this spec, I find it pretty unclear. It is doing a lot of implicit asserts. `it` blocks should end with an expect statement, but there is an expert then a bunch of actions which sometime is implicitly testing a journey, and sometime is just to progress to the next bit.

Reply:
I refactored test's description to match the flow of this spec (using same conventions) but yeah, we need to revisit this to make it match all our other specs.